Scripts/RemediationScripts/Remediate-ConfigureConditionalAccessPolicyForPIM.ps1

```diff
@@ -1,43 +1,47 @@
 ﻿<###
 # Overview:
-    This script is used to configure Conditional Access (CA) policy to require PIM elevation from SAW for admin roles (Owner, Contributor, UAA) in a Subscription.
+    This script is used to configure Conditional Access (CA) policy to require PIM elevation from SAW for all built-in roles except a few and all custom roles in a Subscription.
```

Comment:
This script is used to configure Conditional Access (CA) policy to require PIM elevation from SAW for all built-in roles except following: Azure Front Door Domain Contributor, Azure Front Door Domain Reader, Azure Front Door Profile Reader,
      Azure Front Door Secret Contributor, Azure Front Door Secret Reader, Defender for Storage Data Scanner,
      AzureML Compute Operator, Cognitive Services Usages Reader, Key Vault Crypto Service Release User
      
      Please note this script will not configure Conditional Access (CA) policy for custom roles.